src/redist/targets/GenerateInstallers.targets

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <Target Name="GenerateInstallers"
-          AfterTargets="Pack"
+          BeforeTargets="Pack"
```

Comment:
why this change?

Reply:
@rainersigwald
Changing "AfterTargets" to "BeforeTargets" as the last target on the "GenerateInstallers" list was failing silently.